index.js

```diff
@@ -9,7 +9,19 @@ const SSM = require('aws-sdk/clients/ssm');
 
 const { SSM_PARAMETERS_PATH } = process.env;
 
-const ssm = new SSM();
+const ssm = new SSM({
```

Comment:
Could you explain why this is better than exponential backoff that is default sdk retry method with configurable base timeout value?

Reply:
@alapto I don't know if it is better or not, but I would think that it would be better to retry a couple of time right away if there is not much throttling going on it would be faster, and after that ease out.

Reply:
we can try out with the base config if you think that is better

Reply:
@laardee Fixed retries under throttling conditions amplify the throttling problem and thus it is advisable to use exponential backoff to avoid any request amplification from retries. 

For example using AWS default retry method with `base: 300` and max retries of 5, would produce following retry schedule:
1. Wait 300ms
2. Wait 600ms
3. Wait 1200ms
4. Wait 2400ms
5. Wait 4800ms

Totatling to 9300ms wait time.

Reply:
we can go with that